app/helpers/lti_helper.rb

```diff
@@ -0,0 +1,2 @@
+module LtiHelper
```

Comment:
I think you can create a function like `isLTI?` which checks the session and says whether it is an LTI session or not. Similarly, we can have a `setLTI`

Reply:
is it needed for our current functionality ?